spec/examples_spec.lua

```diff
@@ -0,0 +1,32 @@
+-- Copyright (c) 2024, The Pallene Developers
+-- Pallene Tracer is licensed under the MIT license.
+-- Please refer to the LICENSE and AUTHORS files for details
+-- SPDX-License-Identifier: MIT
+
+local util = require "spec.util"
+
+local function assert_example(example, expected_content)
+    local cdir  = util.shell_quote("examples/"..example)
+    local ok, err, output_content, _ =
+        util.outputs_of_execute(string.format("cd %s && make --quiet && ../../pt-lua main.lua", cdir))
+    assert(ok, err)
+    assert.are.same(expected_content, output_content)
+
+    -- With Pallene Tracer tracebacks enabled
+    local ok, err, output_content, _ =
+        util.outputs_of_execute(string.format([[
+            cd %s
+            make clean --quiet
+            make debug --quiet
+            ../../pt-lua main.lua ]], cdir))
+    assert(ok, err)
+    assert.are.same(expected_content, output_content)
```

Comment:
Why are we compiling and asserting twice?

Reply:
The first one is w/o Tracebacks, the second one is with tracebacks. Testing twice to ensure we are not doing anything terribly wrong in Pallene Tracer. But there is a nit, we need to run the first assertion with normal system Lua.

Reply:
Could those be two separate `it`  blocks?